wallet.js

```diff
@@ -30,6 +29,8 @@ let child_hd_node
 let derivation_path
 const toXOnly = pubKey => (pubKey.length === 32 ? pubKey : pubKey.slice(1, 33));
 
+const get_min_sat_utxo_limit = () => process.env.IGNORE_UTXOS_BELOW_SATS || 1001
```

Comment:
Can we validate if it is a valid int value? I mean, can it return `undefined` or string value? So it could be great if we sanitize the env var.
![image](https://github.com/deezy-inc/sat-hunter/assets/126987350/cbac002d-1b78-44ce-b1d5-bedce7913f49)

Reply:
fair point, I added Number, 
however runtime Javascript will parse String to number when it tries to compare to other number, ex:

<img width="118" alt="image" src="https://github.com/deezy-inc/sat-hunter/assets/159653027/1049a149-f754-4f43-a173-6e806ecbc14a">

Reply:
Yes, I know, but it is always better to keep a strict type.